lenses/httpd.aug

```diff
@@ -95,10 +95,19 @@ let arg_wordlist =
 
 let argv (l:lens) = l . (sep_spc . l)*
 
+(* arg_dir_msg may be the last or only argument *)
+let dir_args = (argv (arg_dir|arg_wordlist) . (sep_spc . arg_dir_msg)?) | arg_dir_msg
+
 let directive =
-    (* arg_dir_msg may be the last or only argument *)
-     let dir_args = (argv (arg_dir|arg_wordlist) . (sep_spc . arg_dir_msg)?) | arg_dir_msg
-  in [ indent . label "directive" . store word .  (sep_spc . dir_args)? . eol ]
+  [ indent . label "directive" . store word .  (sep_spc . dir_args)? . eol ]
+
+let perl_directive =
+     let perl_word = word . ("::" . word)* . "->" . word
```

Comment:
Yes, I was thinking of excluding it from the existing section re and handling it specially.